Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvcr.io/nvidia/tritonserver:23.05-py3
+FROM nvcr.io/nvidia/tritonserver:23.06-py3
 
 RUN mkdir /opt/tritonserver/caches/redis

```

Comment:
LGTM, just need to account for existing directory and overwrite now.

Reply:
hah - yes guess we don't have to make that directory anymore.